osu.Game/Overlays/Chat/DrawableChannel.cs

```diff
@@ -79,6 +82,36 @@ private void load()
             Channel.PendingMessageResolved += pendingMessageResolved;
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            highlightedMessage = Channel.HighlightedMessage.GetBoundCopy();
+            highlightedMessage.BindValueChanged(_ => processMessageHighlighting(), true);
+        }
+
+        /// <summary>
+        /// Processes any pending message in <see cref="highlightedMessage"/>.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="CompositeDrawable.ScheduleAfterChildren"/> is for ensuring the scroll flow has updated with any new chat lines.
```

Comment:
Weird to put this in xmldoc. It's an implementation detail so it should be an inline comment.

Reply:
Hmm, I thought we usually leave implementation details in the remarks section. The sole reason I put it there is to benefit from referencing methods, but will move to comment if that's not where it should be placed.